Python/bytecodes.c

```diff
@@ -1364,15 +1378,8 @@ dummy_func(
             null = NULL;
         }
 
-        inst(LOAD_GLOBAL_BUILTIN, (unused/1, index/1, mod_version/1, bltn_version/1 -- null if (oparg & 1), res)) {
```

Comment:
Once we lose this opcode signature, the cache structure becomes a little harder to read from the code. Is it worth a comment?

Reply:
The cache is a set of mostly independent parts, so breaking it into micro-ops is as clear, I think.
Adding comments for each micro-op, as you mentioned below, is probably more useful.

Reply:
On second thoughts, each micro-op already specifies its cache entry, so the comment is likely redundant or wrong.
I've commented on the `SKIP_CACHE` entries, to clarify what is being skipped.